### ISSUE TITLE
Banana shower placement should act more like a drag operation

### ISSUE THREAD

**Issue by peppy** [ppy/osu#13977]
Currently when placing a banana shower, a right click is required to end the placement. This doesn't feel intuitive and threw me off. It should probably behave the same as hold notes in osu!mania's editor, which will end placement on mouse up as a standard drag operation.

**Comment by ekrctb** [comment]
The current implementation is mimic of the standard spinner placement and didn't think about much but I agree dragging is probably better.

**Comment by Darius-Wattimena** [comment]
Is this really something we want? You can now easily create long spinners without having to hold any buttons. Having the ability to drag would probably be useful when you want to place short spinners. Although, short spinners are rarely seen in osu!catch as spinners are most of the time placed at the end of a map.

A hybrid solution could be an option? When holding the left mouse button you can drag, but when clicking the right mouse button you can maintain the 2 click place method (or the other way around). Not sure if that would become counter-initiative.

**Comment by peppy** [comment]
I agree we probably want something implemented which is shared across all hold notes, rather than having two different interactions depending on which ruleset or object you are interacting with. What you mention as the "hybrid solution" could definitely be explored when implementing this.
